services/core-api/tests/now_application_factories.py

```diff
@@ -432,7 +445,25 @@ class Params:
     proposed_start_date = factory.Faker('past_datetime')
     last_updated_date = datetime.utcnow()
     proposed_end_date = factory.Faker('past_datetime')
-
+    directions_to_site = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    security_not_required_reasons = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    security_not_required = True
+    imported_by = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    imported_date = factory.Faker('past_datetime')
+    annual_summary_submitted = False
+    is_first_year_of_multi = False
+    verified_by_user_date = factory.Faker('past_datetime')
+    decision_by_user_date = factory.Faker('past_datetime')
+    req_access_authorization_numbers = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    relationship_to_applicant = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    liability_adjustment = factory.fuzzy.FuzzyInteger(1, 10000)
+    crown_grant_or_district_lot_numbers = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    adjusted_annual_maximum_tonnage = factory.fuzzy.FuzzyInteger(1, 10000)
+    first_aid_cert_level = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+    is_applicant_individual_or_company = factory.Faker('sentence', nb_words=100, variable_nb_words=True)
+
+    access_operations = factory.RelatedFactory(AccessOperationsFactory, 'now_application')
+    access_roads = factory.RelatedFactory(AccessRoadsFactory, 'now_application')
     blasting_operation = factory.RelatedFactory(BlastingOperationFactory, 'now_application')
```

Comment:
What are "access_operations" and "access_roads" supposed to represent? 
This should match the response_model/payload that is returned to the FE and neither of these are keys. 

Can you elaborate on these additions?

Reply:
Chatted in person!